### ISSUE TITLE
Add button to copy translated text in popup on page and text translation tab

### ISSUE THREAD

**Issue by vitonsky** [translate-tools/linguist#301]

**Comment by Flashwalker** [comment]
Yeah, it's necessary. Especially for those sites/apps that block the right click.

**Comment by Prasanta-Hembram** [comment]
It can be difficult to copy text inside the linguist text translation tab. A copy button would make it much easier to do so.

**Comment by vitonsky** [comment]
Ok, it looks we have demand on this feature. Will be implemented in next major release

**Comment by vitonsky** [comment]
@Prasanta-Hembram @Flashwalker @Terkyz could you suggest where to place a button to copy text.
You can just explain or draw a rectangle on my screenshot in your answers.

![image](https://github.com/translate-tools/linguist/assets/86191922/05377834-f72a-43f5-93c5-c1e3a8273d90)

**Comment by Flashwalker** [comment]
Variant 1

![v1](https://github.com/translate-tools/linguist/assets/2188215/77f128cd-38d6-4796-87fe-5ad3f5df86a4)


Variant 2 - button on mouse hover over the translated text

![v2](https://github.com/translate-tools/linguist/assets/2188215/d617276c-3181-443e-b2c2-f6dad159dcbf)

**Comment by realgooseman** [comment]
I redesigned the pop-up window to make it cleaner and added a copy to clipboard button, take a look at #397

**Comment by vitonsky** [comment]
@Flashwalker @Prasanta-Hembram check the UI in comment https://github.com/translate-tools/linguist/issues/397#issuecomment-1705454346

If you have thoughts about it, share it with us on #397
